### ISSUE TITLE
Fieldtype Media required all Image Metadata

### ISSUE THREAD

**Issue by kvbjam** [Gernott/mask#548]
We have made an update to version 8.1.0 and now all additional fields should be filled for the content type Media.
Typo3: 11.5.26

![Edit-Page-Content](https://user-images.githubusercontent.com/88367683/236240068-ec593204-9853-4932-8f9e-fe2597b42300.png)

**Comment by nhovratov** [comment]
This has nothing to do with the Mask update. It's as TYPO3 core bug. If you define e.g. `minitems => 2` and you only add one item, all fields get marked as required. As soon as the second item is added, they dissappear again.

So, I'm pretty sure I saw this behaviour at some point. If I can't find a related bug report, I will create a new issue in TYPO3 forge.

Thanks nevertheless for reporting, eventhough I can't fix this in Mask! We already found a bunch of TYPO3 core bug through the usage of Mask.
